packages/bsky/src/db/periodic-moderation-action-reversal.ts

```diff
@@ -0,0 +1,78 @@
+import { wait } from '@atproto/common'
+import { Leader } from './leader'
+import { dbLogger } from '../logger'
+import Database from '.'
+import AppContext from '../context'
+
+export const MODERATION_ACTION_REVERSAL_ID = 1011
+
+export class PeriodicModerationActionReversal {
+  leader = new Leader(MODERATION_ACTION_REVERSAL_ID, this.db)
+  destroyed = false
+
+  constructor(public db: Database, private appContext: AppContext) {}
```

Comment:
I think this may have been intended to change to:
```suggestion
  constructor(private appContext: AppContext) {}
```

Reply:
🤦🏽‍♂️